src/server/conn_context.h

```diff
@@ -24,6 +24,9 @@ class StoredCmd {
  public:
   StoredCmd(const CommandId* cid, CmdArgList args);
 
+  // Create on top of already filled tightly-packed buffer.
+  StoredCmd(std::string&& buffer, const CommandId* cid, CmdArgList args);
```

Comment:
why not `std::string` without `&&` ?

Reply:
Yes, a plain string works as well. It just cuts off any possibilities of reading the arguments (see previous comment), but its not needed anyways